README.md

```diff
@@ -54,9 +54,15 @@ as you like in this plugins. The most important choices are:
 To test/develop the kingscode scaffold you can locally invoke this plugin.
 create a test project (`vue create scaffold-tester`) just like above.
 
+browse to the scaffold-tester folder, which was just created.
+
+install vuetify
+
+`vue add vuetify`
+
```

Comment:
there is much more then just vuetify, So should this line be here?

>just like above.
my referer to the installation is not really notable, perhaps just change that

Reply:
It is the full instruction to locally test the cli:
![image](https://user-images.githubusercontent.com/47124570/112859071-c6deca00-90b2-11eb-99d9-04777819434d.png)

Reply:
No it isn't. Just like i said before there is much more then vuetify.

vue router
vuex
dart sass
..
..

For development you need to follow the instructions "just like above"
You could add those instructions just like you've added vuetify again at this place. But then you should remove the "just like above"
When you do that we'd have to maintain that at two places so i'm opting to revert your change and just make the line "just like above" more notable.

Something like this:

```
### Development
- To test/develop the kingscode scaffold you can locally invoke this plugin.
create a test project (`vue create scaffold-tester`) just like in the Installation part, you can follow those instructions and continue here.

- install the cli-plugin locally

`npm install path\to\local\vue-cli-plugin-kingscode-scaffold`

- invoke the plugin

`vue invoke kingscode-scaffold`
```

Reply:
I'm not sure why you keep pushing. The only fact is, Vuetify is the only one mandatory for this cli.

Vue router, vuex and even dart sass (the dart sass thing is not required, if you don't select css pre-processors), are optional.

Reply:
Let's call about this.
BTW PR was already approved so i'm not pushing but in my opinion this adds more confusion instead of removing it.